com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/helpers/ApplicationColorDetector.java

```diff
@@ -26,8 +27,6 @@
 @Named("applicationColorDetector")
 public class ApplicationColorDetector {
 
-    private static final ApplicationColor COLOR_OF_APPLICATIONS_WITHOUT_SUFFIX = ApplicationColor.BLUE;
```

Comment:
Why this is removed from here?

Reply:
It was moved from cf.process in cf.core package because the constant couldn't be used from classes in cf.core package.